src/lib/Components/Consignments/Screens/FinalSubmissionQuestions.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from "react"
+
+import { LayoutAnimation, NavigatorIOS, Route, ScrollView, View, ViewProperties } from "react-native"
+import ConsignmentBG from "../Components/ConsignmentBG"
+import { BodyText, LargeHeadline, Subtitle } from "../Typography"
+
+import { ArtistResult, ConsignmentSetup } from "../"
+import TODO from "../Components/ArtworkConsignmentTodo"
+import Text from "../Components/TextInput"
+import Toggle from "../Components/Toggle"
+import Artist from "./Artist"
+import Provenance from "./Provenance"
+import SelectFromPhotoLibrary from "./SelectFromPhotoLibrary"
+import Welcome from "./Welcome"
+
+interface Props extends ViewProperties {
+  navigator: NavigatorIOS
+  route: Route // this gets set by NavigatorIOS
+  setup: ConsignmentSetup
+}
+
+export default class FinalSubmissionQuestions extends React.Component<Props, ConsignmentSetup> {
+  constructor(props) {
+    super(props)
+    this.state = props.setup
+  }
+
+  submitWork = () => {
+    // OK
+  }
+
+  updateEdition = () => {
+    // React Native's Typings are wrong here, I want to pass in
+    // no arguments.
+    const animate = LayoutAnimation.easeInEaseOut as any
+    animate()
+
+    this.setState({
+      editionInfo: this.state.editionInfo ? null : {},
+    })
+  }
+
+  updateSigned = () => this.setState({ signed: this.state.signed ? false : true })
```

Comment:
Could you do `{ signed: !Boolean(this.state.signed) }`? (or just `!this.state.signed` if it's coming back as truthy already)